src/components/SongList/SongList.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react'
```

Comment:
separated some of the state logic `SongList/index.js` from rendering logic `SongList/SongList.js`. ideally, we can pull the remaining state in `SongList.js` into another redux slice/reducer.